### DISCUSSION TITLE
Found 5783 templates loaded with deprecated protocol syntax warning

### DISCUSSION THREAD

**Discussion by petruuux** [projectdiscovery/nuclei#3665] [Q&A] [answered]
I was trying to scan a list of domain, and before the scan started it gave me this warning: 
[WRN] Found 5783 templates loaded with deprecated protocol syntax, update before v2.9.5 for continued support.
(screenshot on: https://imgur.com/Z7Nj9YE.png)
Why does this happen? How can i fix it? I read a recent discussion about it and it says that it only happens with custom/private templates, but i don't use any.

**Comment by MetzinAround** [comment, marked as answer]
That warning is part of a change the structure of templates is undergoing. It's only a warning and not an error right now, so you're in the clear. 
https://github.com/projectdiscovery/nuclei/discussions/3648

**Reply by petruuux** [reply]
Thank you. So i don't have to update anything, right?

**Reply by ehsandeep** [reply]
@petruuux no, you don't
